### DISCUSSION TITLE
SSLError, trying to install packages (kivy) on python 1.12

### DISCUSSION THREAD

**Discussion by Idan0g** [python-poetry/poetry#9137] [General]
I tried to use: pip install kivy, ran into this error: ERROR: Could not install packages due to an OSError: HTTPSConnectionPool(host='files.pythonhosted.org', port=443): Max retries exceeded with url: /packages/f2/95/79dce0bbfb66895c68f0844d54212149665b8b1d5e1247151fa9514055d9/Kivy-2.3.0-cp312-cp312-win_amd64.whl.metadata (Caused by SSLError(CertificateError("hostname 'files.pythonhosted.org' doesn't match 'r.shared-319-default.ssl.fastly.net'"))). 
and I cant find a solution.

**Comment by dimbleby** [comment]
> I tried to use: pip install kivy

you are in the wrong place, this is the poetry repository
